tests/contenttypes_tests/test_fields.py

```diff
@@ -33,3 +33,9 @@ def test_value_to_string(self):
         answer2 = Answer.objects.create(question=question)
         result = json.loads(Question.answer_set.field.value_to_string(question))
         self.assertCountEqual(result, [answer1.pk, answer2.pk])
+
+    def test_filter_on_related_proxy_model(self):
+        self.assertEqual(
```

Comment:
It'd be a bit nicer to verify the result of some query. I'm not sure if we have any tests like this.